LeaderboardBackend/Migrations/20240722190611_RunAddUserId.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace LeaderboardBackend.Migrations
+{
+    /// <inheritdoc />
+    public partial class RunAddUserId : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Guid>(
+                name: "user_id",
+                table: "runs",
+                type: "uuid",
+                nullable: false,
+                defaultValue: new Guid("00000000-0000-0000-0000-000000000000"));
```

Comment:
Should this provide a default value when its not nullable?

I can't remember the specific order of operations, but this may return an fkey error instead of a null value check error when interacting with the database.

If the intention is to allow records without a user, just make it nullable (imo)

Reply:
That's auto-generated. I don't think postgres allows not having any default value. It'll still violate the FK constraint so I'm not worried.

Reply:
I'd imagine we'll set the relation to null if the user's been deleted, right? We need to factor that in. Unless we delete the run, as well

Reply:
There's no current plan for actually deleting user records. We can just ban them. Even if there was, we could just use soft deletes like for lbs, cats, and runs.